ADOPTERS.MD

```diff
@@ -31,5 +31,6 @@ If you would like to be included in this table, please submit a comment to [this
 | VMware                             | Cloud Native Runtimes for VMware Tanzu                                 | Generally Available      | [link](https://network.tanzu.vmware.com/products/serverless)         |
 | VMware                             | VMware Event Broker Appliance                                          | Development              | [link](https://vmweventbroker.io/)                                   |
 | vorteil pty ltd*                   | Serverless orchestration engine                                        | Development & Production | [link](http://vorteil.io)                                            |
+| HighCohesion                       | SaaS E-commerce Integration Platform                                   | Development              | [link](https://www.highcohesion.com/)                                |
```

Comment:
Can you insert in alphabetically order? it would be after the row `IBM`

Reply:
Well, just wait a minute, if we want it alphabetically (and we do), how does it come _after_ IBM @csantanapr ?